pos/app/components/molecules/InputComment.tsx

```diff
@@ -0,0 +1,34 @@
+import type { WithId } from "common/lib/typeguard";
+import type { OrderEntity } from "common/models/order";
+import { useState } from "react";
+import { Input } from "../ui/input";
+
+type props = {
+  order: WithId<OrderEntity>;
+  mutateOrder: (order: WithId<OrderEntity>, descComment: string) => void; // これをコンポーネントの中で呼び出す
+};
+
```

Comment:
自分で書いておいて申し訳ないんですけど、ここで order の何かを表示しているわけではないので、order は取っちゃいたいですね
```suggestion
type props = {
  addCommentToOrder: (comment: string) => void;
};

```